src/w3c/headers.js

```diff
@@ -95,6 +95,7 @@ import { concatDate, joinAnd, ISODate } from "core/utils";
 import hb from "handlebars.runtime";
 import { pub } from "core/pubsubhub";
 import tmpls from "templates";
+import { w3cDefaults } from "w3c/defaults";
```

Comment:
Just realized that this is imported but never used, right?

Reply:
Oops, I used it once and removed later, but forgot about this. Good catch!